ophyd/signal.py

```diff
@@ -64,7 +64,7 @@ def _repr_info(self):
         yield ('value', self.value)
         yield ('timestamp', self.timestamp)
 
-    def get(self):
+    def get(self, **kwargs):
```

Comment:
because sub-classes do and we in various places in PVPositioner we pass key words in.  When mocking things out with soft signals you need to be able to accept the extra (unused) inputs.